src/scripts/worldmap/MapHelper.ts

```diff
@@ -218,8 +218,11 @@ class MapHelper {
         }
 
         const challengeActive = App.game.challenges.list.requireCompletePokedex.active();
+        const shinyChallengeActive = App.game.challenges.list.requireCompleteShinyPokedex.active();
         const nextStartingTownUnlocked = TownList[GameConstants.StartingTowns[player.highestRegion() + 1]]?.isUnlocked() ?? false;
-        const fullDex = AchievementHandler.findByName(`${GameConstants.camelCaseToString(GameConstants.Region[player.highestRegion()])} Master`).isCompleted();
+        const fullDex = AchievementHandler.findByName(
+            `${GameConstants.camelCaseToString(GameConstants.Region[player.highestRegion()])} ${(shinyChallengeActive ? 'Shiny' : '')} Master`
```

Comment:
This and the ProfNPC one are broken, because `shinyChallengeActive = false` means we have two spaces in this string where we had one before, so the achievement can't be found.

Causes the game to crash for me with this when loading:
```
Uncaught (in promise) TypeError: Unable to process binding "css: function(){return { flash:MapHelper.ableToTravel()} }"
Message: AchievementHandler.findByName(...) is undefined
```

Reply:
Yeah, that was not smart. I have fixed it now, so now you can talk with Oak again